store/commitment/store.go

```diff
@@ -428,7 +444,7 @@ loop:
 				chStorage <- &store.KVPair{
 					Key:      node.Key,
 					Value:    node.Value,
-					StoreKey: storeKey,
+					StoreKey: storeKey, // TODO check if this is valid
```

Comment:
@cool-develope this looks to be empty in the flow, what is the reason we are setting an empty storekey here